packages/vm/test/api/copy.spec.ts

```diff
@@ -18,12 +18,6 @@ describe('VM Copy Test', () => {
       'account exists before copy',
     )
 
-    const vmCopy = await vm.shallowCopy()
-    assert.isUndefined(
-      await vmCopy.stateManager.getAccount(address),
-      'non-committed checkpoints will not be copied',
```

Comment:
I'm not entirely sure why this is deleted, am I missing something?

Reply:
The behavior just changed here, non-committed checkpoints *are* now copied in the default case of non caches used (which is ok, the checkpointing is in this case fully done by the trie checkpointing mechanism).

Since this check is not testing a required property but just a it-was-like-this-in-this-case property which now changed I removed the test (the alternative would have been to setup a full with-caches-without-caches setup respetively introduce some differentiation here, which I totally found overblown - again, since this is just not testing something which needs to have the tested property.

Hope this is clear. 🙂